src/locales/default.json

```diff
@@ -13,7 +13,8 @@
         "pools-fetch-error": "Failed to fetch pools.",
         "price-fetch-error": "Our pricing provider, CoinGecko, is currently down.",
         "retry-label": "Refresh to try again",
-        "boostedPools": "Increased yields, greater capital efficiency, deeper liquidity. Try Balancer Boosted pools now →"
+        "boostedPools": "Increased yields, greater capital efficiency, deeper liquidity. Try Balancer Boosted pools now →",
+        "vebalL2": "As we transition to the veBAL system, liquidity mining incentives on Polygon and Arbitrum are suspended. Stake to start earning BAL as soon as distributions resume on April 28th."
```

Comment:
@pkattera this is the copy I could use your help with. Feel free to commit to this PR when you have a better version for this. Thanks.

Reply:
@markusbkoch I have just done a new commit with updated copy.